scripts/linking.py

```diff
@@ -448,8 +525,8 @@ def eval_candidate_generation_and_linking(examples: List[data_util.MedMentionExa
     candidate_generator: CandidateGenerator
         A CandidateGenerator instance for generating linking candidates for mentions
     k_list: List[int]
-        A list of k values determining how many candidates are generated
-    thresholds: List[float]
+        A list of values determining how many candidates are generated.
+    thresholds: float
```

Comment:
types here are now inconsistent with the parameters

Reply:
see above